src/consts/models.tsx

```diff
@@ -261,6 +261,11 @@ export enum ConfirmAddressFlowType {
   UNSUBSCRIBE = 'UNSUBSCRIBE',
 }
 
+export enum NotificationApiErrorMessages {
```

Comment:
we have root/errors/messages for this

Reply:
I see now and I don't know why part of consts are stored there.